packages/suite/src/components/wallet/DiscoveryProgress.tsx

```diff
@@ -6,7 +6,7 @@ import { zIndices } from '@trezor/theme';
 
 const StyledProgressBar = styled(ProgressBar)`
     height: 0;
-    z-index: ${zIndices.discoveryProgress};
+    z-index: ${zIndices.modal};
```

Comment:
Not sure about this

![image](https://github.com/trezor/trezor-suite/assets/152899911/88425fba-31ea-4078-ba1f-26624127af0e)